ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-deprecated-nonprod.json

```diff
@@ -0,0 +1,44 @@
+[
```

Comment:
maybe we can delete this file, these events are not live so I don't think there is much value in maintaining them, it would allow us to see case event history for placement in aat/demo but I don't think it's too important. can be removed later in separate PR if this one is already in testing/nearly done

Reply:
I think we can leave it in as we made changes to [AuthorisationCaseEvent-nonprod.json](https://github.com/hmcts/fpl-ccd-configuration/blob/dc5f4dae3e051eeae89f590621523e7be2113214/ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json)

Reply:
Sure, we can leave it 👍